opsdroid/locale/hi/LC_MESSAGES/opsdroid.po

```diff
@@ -303,11 +303,10 @@ msgid "Started opsdroid %s"
 msgstr "opsdroid %s शुरू"
 
 #: opsdroid/matchers.py:127
-msgid ""
-"Recast.AI is now called SAP Conversational AI, this matcher  will stop "
```

Comment:
This line needs to stay the same, that's why the tests are failing. Our localization expects the strings to be literal ones from the code base. 

The previous PR that added the Hindi translation, probably followed the steps for translating opsdroid.